tests/ConditionalLogicTest.php

```diff
@@ -24,11 +24,11 @@ class ConditionalLogicTest extends TestCase
 {
     public function testConditionalLogic()
     {
-        $conditionalLogic = [
+        $conditionalLogic = [[
             'field' => 'field_f5456193',
             'operator' => '==',
             'value' => 10,
-        ];
+        ]];
```

Comment:
Since you changed this, it means this update isn't backwards compatible?

Reply:
This modification has no impact on the operation. The table level is no longer managed at the get() level but in the class that centralizes the conditions. The table generated for ACF remains compatible.